_data/simple-icons.json

```
@@ -3400,6 +3405,11 @@
             "hex": "FF0000",
             "source": "https://www.youtube.com/yt/about/brand-resources/#logos-icons-colors"
         },
+        {
+            "title": "Zalando",
+            "hex": "FF6900",
+            "source": "https://www.zalando.de/"
```

```suggestion
            "source": "https://www.zalando.co.uk/"
```

So that the website is in English rather then German :wink: